<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbOp1">
     <property name="text">
      <string>First Matrix</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lbOpt">
     <property name="text">
      <string>Operation</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lbOp2">
     <property name="text">
      <string>Second Matrix</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="cbOp1"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbOpt"/>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="cbOp2"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbRes">
     <property name="text">
      <string>Result</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTableView" name="tvRes"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="pbCal">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
